src/Tribe/Attendee_Registration/Main.php

```diff
@@ -53,6 +53,43 @@ public function is_on_page() {
 		return ! empty( $wp_query->query_vars[ $this->key_query_var ] ) ;
 	}
 
+	/**
+	 * Returns whether or not the user is on a page using the attendee registration shortcode
+	 *
+	 * @return boolean
+	 */
+	public function is_using_shortcode() {
+		global $wp_query;
+
+		return ! empty( $wp_query->queried_object->post_content ) && has_shortcode( $wp_query->queried_object->post_content, 'tribe_attendee_registration' );
+	}
+
+	/**
+	 * Returns a list of providers in the "cart" (AR page)
+	 *
+	 * @return array
+	 */
+	public function providers_in_cart() {
+		$providers = apply_filters( 'tribe_providers_in_cart', [] );
```

Comment:
Docblock

Reply:
```suggestion
		/**
		 * Allow filtering of commerce providers in cart.
		 *
		 * @since TBD
		 *
		 * @param array $providers List of commerce providers in cart.
		 */
		$providers = apply_filters( 'tribe_providers_in_cart', [] );
```